evergreen.yaml

```diff
@@ -168,31 +168,20 @@ post:
 #           Buildvariants             #
 #######################################
 buildvariants:
-  - name: race-detector
-    display_name: Race Detector (Arch Linux)
-    expansions:
-      GOROOT: /opt/golang/go1.16
-      RACE_DETECTOR: true
-    run_on:
-      - archlinux-new-small
-      - archlinux-new-large
-    tasks:
-      - ".test"
 
   - name: lint
-    display_name: Lint (Arch Linux)
+    display_name: Lint (Ubuntu 22.04)
```

Comment:
We can just call this `Lint`, since the particular Linux distro isn't that relevant.

Reply:
Fixed